docs/platform/nodebalancer/force-all-connections-on-ssl-with-nodebalancers

```diff
@@ -0,0 +1,356 @@
+---
+author:
+  name: Joel Kruger
+  email: jkruger@linode.com
+description: 'Forcing all connections to use SSL with NodeBalancers.'
+keywords: 'Linode,NodeBalancer,SSL,redirect'
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+modified: Monday, May 18th, 2015
+modified_by:
+  name: Joel Kruger
+published: 'Monday, May 25th, 2015'
+title: Forcing SSL connections with Nodebalancers
+---
+
+The following documentation will assist you with obtaining a valid, commercially signed SSL certificate and installing it on your NodeBalancer. This guide provides step-by-step instructions for configuring a NodeBalancer to redirect all web connections, using SSL, over port 443/HTTPS. Instructions will be provided to configure this with both the Apache and Nginx servers on Debian and Redhat based distributions.
+
+{: .note }
+> 
+> Please note that commercial SSL certificates require a unique IP address for each certificate. As a result, you will only be able to host one SSL-enabled website, per NodeBalancer
```

Comment:
Is it not correct that this can only be done when terminating SSL at the Backend Nodes? That is to ask, a NodeBalancer can only support 1 SSL certificate/chained certificate?